### ISSUE TITLE
RichEditor error when add attachment file

### ISSUE THREAD

**Issue by budiardianata** [laravel-filament/filament#748]
### Package

filament/forms

### Package Version

v 2.4

### Laravel Version

v 8.73

### Livewire Version

v 2.8

### PHP Version

8.0

### Bug description

[RichEditor](https://filamentadmin.com/docs/2.x/forms/fields#rich-editor) error when add attachment file

![image](https://user-images.githubusercontent.com/17608807/144359432-5d0d85b8-a787-407b-ac20-73ebfb204946.png)

### Relevant log output

```shell
Cannot assign string to property App\Http\Livewire\ArticleForm::$componentFileAttachments of type array
```

**Comment by karam-mustafa** [comment]
Are you disable the attachFiles in toolbarButtons ? 
Can you take a pic of "54" tab number please?

**Comment by budiardianata** [comment]
I not disable toolbar. 

This is my RichEditor form.
```php
RichEditor::make('content')
    ->fileAttachmentsDirectory('attachments')
    ->fileAttachmentsVisibility('public')
```

**Comment by karam-mustafa** [comment]
Ok checking!

**Comment by budiardianata** [comment]
For info, i try to edit this line on my  project

 [public array $componentFileAttachments = []  ](https://github.com/laravel-filament/filament/blob/a39bbca51f47e2f37bbebbae53107d34a2f5df2d/packages/forms/src/Concerns/InteractsWithForms.php#L15)

To 
```php
public string|array $componentFileAttachments = []
```

And this issue is solved

**Comment by karam-mustafa** [comment]
All the attachments work fine for me.

I used this code in form function:
`Components\RichEditor::make('content')->attachmentDirectory('attachments')`

**Comment by danharrin** [comment]
I can't replicate it either... and I really wouldn't modify the property like that, a string shouldn't be assigned to that property...

**Comment by danharrin** [comment]
Closing, if anyone else is able to replicate this please @ me and I'll reopen.

**Comment by danharrin** [comment]
Fixed by #787.

**Comment by danharrin** [comment]
This has now been released in Livewire v2.8.2. Please update :)
